Source/Core/Core/CMakeLists.txt

```diff
@@ -343,6 +343,7 @@ if(WIN32)
     setupapi.lib
     iphlpapi.lib
   )
+  target_compile_definitions(core PRIVATE /D _WINSOCK_DEPRECATED_NO_WARNINGS) 
```

Comment:
This should be `target_compile_definitions(core PRIVATE "-D_WINSOCK_DEPRECATED_NO_WARNINGS")` (another source of rogue -D/D in command line)